### PR TITLE
shared: codemod `shared/components/FileMatchChildren` `shared/components/Markdown` `shared/components/activation` components to CSS modules

### PR DESCRIPTION
**Pull request by gitstart-sourcegraph**
## Descriptions

This PR results from the [global-css-to-css-modules](https://github.com/sourcegraph/codemod/tree/main/src/transforms/globalCssToCssModule) codemod application with some manual fixes, caused by the usage of global CSS classes tied to a React component in other parts of the codebase

## Refs

- [SG Issue](https://github.com/sourcegraph/sourcegraph/issues/27078)
- [Gitstart Ticket](https://app.gitstart.com/clients/sourcegraph/tickets/SG-27078)

## Success criteria

The stylesheets in the following directories are converted to CSS modules.

- client/shared/src/components/FileMatchChildren
- client/shared/src/components/Markdown
- client/shared/src/components/activation

### CODE REVIEW

File: client/shared/src/components/Markdown.module.scss [hunk 1]
```diff
@@ -77,20 +77,4 @@
     img {
         max-width: 100%;
     }
-
-    &.search-result-match__markdown {
```
Comment (limitedmage) [inline client/shared/src/components/Markdown.module.scss:81]: Is this style not needed anymore? (Or did I miss where it moved to?)

**Review by limitedmage — APPROVED** [review]

File: client/shared/src/components/Markdown.module.scss (hunk 1 above)
Comment (valerybugakov) [reply client/shared/src/components/Markdown.module.scss:81]: @limitedmage, thanks for noticing it! I deleted this class from the markup [here](https://github.com/sourcegraph/sourcegraph/pull/24664/files#diff-9a7226762dd9962a4334fb176e9f7ca3112d0d7539379095f5cd7afc5a208782L166-R172). I tried adding it back now and checked the UI — it doesn't seem to affect the interface (probably that's why I deleted it in the first place). Let me know if you have a use case for these styles in mind.

**Review by valerybugakov — APPROVED** [review]

File: client/shared/src/components/Markdown.module.scss (hunk 1 above)
Comment (gitstart-sourcegraph) [reply client/shared/src/components/Markdown.module.scss:81]: @limitedmage Wait for your feedback so we can merge this PR

Comment (limitedmage) [reply client/shared/src/components/Markdown.module.scss:81]: Looks good! Thanks for double-checking!

**Pull request merged**